Backend/routes/retailerOrderRoutes.js

```
@@ -0,0 +1,62 @@
+    res.json(orders);
+  } catch (error) {
+    res.status(500).json({ error: "Failed to fetch retailer orders" });
+  }
+});
+
+// Add a new retailer order
+router.post("/", async (req, res) => {
+  try {
+    const { product, category, quantity, price } = req.body;
+    // Calculate total
+    const priceNum = parseFloat(price.replace("$", ""));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)